scripts/infra/cloud-instance.sh

```diff
@@ -199,7 +199,7 @@ ibm__stop() {
 
 ibm__start() {
     handle_help_and_instance_name_opts "$@"
-    ibmcloud is instance-start -f "$INSTANCE_NAME"
+    ibmcloud is instance-start "$INSTANCE_NAME"
```

Comment:
@danmcp just as a note I removed this as the `-f` flag isn't present for `instance-start`

Reply:
Thanks!